src/composables/createPost.js

```diff
@@ -0,0 +1,28 @@
+import { reactive, ref } from "vue";
+import { userPostStore } from '../store/post-store.js';
+
+const store = userPostStore();
+
+const { addPosts } = store;
+export const createPostApi = () => {
+    const postDetails = reactive({
+        title: "",
+        photo: null,
+        description: "",
+    });
+    const isLoading = ref(false)
+    const sucessModal = ref(false)
+
+    const createPostsByUser = async () => {
```

Comment:
createPostsByUser => by user? who else going to create the post
function name should be only createPost
any why posts, `s` its not plural or multiple @ShivanshSinghnnovative